app/javascript/source/javascript/binx_admin.js

```diff
@@ -13,4 +15,17 @@ window.BinxAdmin = class BinxAdmin {
         .css("max-width", "100%");
     }
   }
+  initGraphs() {
+    if (window.location.href.match('\\admin/graphs')){
+      window.onload = function(e) {
```

Comment:
You don't need this `window.onload` function - `initGraphs` is only called when page has finished initializing.